test/house_test.rb

```diff
@@ -0,0 +1,70 @@
+
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/room'
+require './lib/house'
+
+class HouseTest < Minitest::Test
+
+  def setup
+    @house = House.new("$400000", "123 sugar lane")
+  end
+
+  def test_it_exists
+    assert_instance_of House, @house
+  end
+
+  def test_it_has_a_price
+    assert_equal "$400000", @house.price
+  end
+
+  def test_it_has_an_address
+    assert_equal "123 sugar lane", @house.address
+  end
+
+  def test_it_initializes_with_empty_rooms_array
+    assert_equal [], @house.rooms
+  end
+
+  def test_it_can_add_rooms_and_store_in_rooms
+    @room_1 = Room.new(:bedroom, 10, '13')
+    assert_equal [@room_1], @house.add_room(@room_1)
+    @room_2 = Room.new(:bedroom, 11, '15')
+    assert_equal [@room_1, @room_2], @house.add_room(@room_2)
+    assert_equal [@room_1, @room_2], @house.rooms
+  end
+
+  def test_it_is_above_market_average
+    assert_equal false, @house.above_market_average?
+  end
+
+  def test_can_return_rooms_in_category
+    @room_1 = Room.new(:bedroom, 10, '13')
+    @room_2 = Room.new(:bedroom, 11, '15')
+    @room_3 = Room.new(:living_room, 25, '15')
+    @room_4 = Room.new(:basement, 30, '41')
+    @house.add_room(@room_1)
+    @house.add_room(@room_2)
+    @house.add_room(@room_3)
+    @house.add_room(@room_4)
+    assert_equal [@room_1, @room_2], @house.rooms_from_category(:bedrooms)
```

Comment:
This is a great test (other than the improper use of the `@room_1, @room_2, etc` instance variables.

Reply:
Ah I see your issue here, You're passing `:bedrooms` as an argument instead of `:bedroom`. This is where copy and pasting directly from the interaction pattern can help you avoid silly mistakes that cause you to get stuck!